src/data/user/Setting.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @fileoverview This file contains all Settings related functions that are triggered by a Customer
+ */
+
+import axios from 'axios';
+import endpoints from '../Endpoint';
+import camelize from 'camelize';
+import { apiResponseErrorHandler } from '../utility/Util';
+import { camelToSnake } from '../utility/ChangeCase';
+
+/** Retrieve Customer"s Settings
+ * @param {string} token
+ * @return {Promise} settings object
+ */
+
+export const getUserSettingMyOrdersColumnAsync = async (token) => {
+  try {
+    const res = await axios({
+      method: 'GET',
+      url: endpoints.API_V3.USER_SETTING_MY_ORDERS_COLUMN,
+      headers: { Authorization: `Bearer ${token}` },
+    });
+    return {
+      ...res,
+      data: camelize(res.data.data),
+    };
+  } catch (e) {
+    return apiResponseErrorHandler(e);
+  }
+};
+
+/** Updates the customer's settings.
+ * @param {string} token
+ * @param {int} id
+ * @param {Object} data
+ * @return {Promise} settings object
+ */
+
+export const putUserSettingMyOrdersColumn = async (token, data) => {
```

Comment:
can we put token at the end of the params same as all of our existing sdk functions